examples/plugins/afk.js

```diff
@@ -0,0 +1,50 @@
+// Plugins are loaded using the bot.loadPlugin() function
+// The load plugin function excepts one function as its only parameter
+// The plugin runs the function passing it two parameter
+// The former is the bot object and the latter is the bot option
+// the options contain version player info and other plugins
+// Here is an example AFK plugin
+
+// Each plugin should have a inject function
+// This is the function that will be put in bot.loadPlugin
+function inject (bot, option) {
+// It is a good idea to make a namespace for your plugin
+  bot.afk = {}
+
+  let afkInterval, rotation
+  // This namespace can be used to store some variuabels for user reffrence.
+  bot.afk.status = 'Idel'
+
+  // All your plugin functions should be in the name space other than the logic.
+  bot.afk.start = async function () {
+    afkInterval = setInterval(async () => {
+      if (rotation === 0) {
+        await bot.look(0, 0)
+        rotation = true
+      } else {
+        await bot.look(Math.PI, 0)
+        rotation = false
+      }
+    }, 3000)
+    bot.afk.status = 'afk'
+  }
+
+  bot.afk.stop = function () {
```

Comment:
() => {

Reply:
Doesn't matter 
in fact, it would be better this way because function () {}
doesn't inherent anything form the higher scope other than the var

Reply:
why would it be better lol, thats old syntax but ig it doesnt matter